pkgs/development/compilers/llvm/common/compiler-rt/default.nix

```diff
@@ -128,10 +128,15 @@ stdenv.mkDerivation ({
   '') + ''
     substituteInPlace lib/builtins/int_util.c \
       --replace "#include <stdlib.h>" ""
+  '' + lib.optionalString (useLLVM && !haveLibc && !stdenv.hostPlatform.isFreeBSD) ''
```

Comment:
This doesn't feel right. If we don't have a libc, then how will we have an `<assert.h>`?